lighthouse-cli/test/cli/__snapshots__/cli-flags-test.js.snap

```diff
@@ -413,9 +456,6 @@ Object {
   "extraHeaders": Object {
     "X-Men": "wolverine",
   },
-  "extraheaders": Object {
-    "xMen": "wolverine",
-  },
```

Comment:
seems the older yargs was overzealous in how it interprets provided config files.